domains/kaseax.json

```diff
@@ -0,0 +1,13 @@
+{
+  "owner": {
+    "username": "Kaseax",
+    "email": "greenpepper1489@gmail.com",
+    "twitter": "kaseaxx",
+    "discord": "Kaseax#0864"
+  },
+  "record": {
+    "A": [
+      "37.114.32.204",
+      "37.114.32.205"
```

Comment:
```suggestion
      "37.114.32.205"
    ]
```

Reply:
There are two different servers. why do you want that?

Reply:
> There are two different servers. why do you want that?

You need the `]` to end the array...

Reply:
oh, im sorry.